crates/sui-graphql-rpc/schema.graphql

```diff
@@ -3399,6 +3400,10 @@ type Query {
 	- `author` is the address of the signer of the transaction or personal msg.
 	"""
 	verifyZkloginSignature(bytes: Base64!, signature: Base64!, intentScope: ZkLoginIntentScope!, author: SuiAddress!): ZkLoginVerifyResult!
+	"""
```

Comment:
nit: can you move the implementation of this up near the other accessor functions so it shows up in that order in the schema? Let's put multi-gets between the point lookups and the rich paginated queries (so at the moment, between `fn transaction_block` and `fn coins`.

Reply:
Moved